app/assets/stylesheets/global.sass

```diff
@@ -22,19 +22,9 @@ a
     text-decoration: none
 
 nav.navbar, .footer
-  background: rgb(37, 42, 58)
```

Comment:
why these changes?  I noticed that the footer is a different shade now than it was before.

Reply:
I originally wanted to just use the `.bg-dark` class from bootstrap. But we really should do bigger style changes in a more cohesive project, so I changed it back.